client/dashboard/emails/resources/mailbox-icon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export type MailboxIconProps = React.SVGProps< SVGSVGElement > & {
+	size?: number | string;
+};
+
+const MailboxIcon: React.FC< MailboxIconProps > = ( { size = 16, ...props } ) => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 16 16"
+		width={ size }
+		height={ size }
+		aria-hidden={ props[ 'aria-label' ] ? undefined : true }
+		focusable="false"
+		{ ...props }
+	>
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M2 1.5H14C14.1326 1.5 14.2598 1.55268 14.3536 1.64645C14.4473 1.74021 14.5 1.86739 14.5 2V9H10C10 9.53043 9.78929 10.0391 9.41421 10.4142C9.03914 10.7893 8.53043 11 8 11C7.46957 11 6.96086 10.7893 6.58579 10.4142C6.21071 10.0391 6 9.53043 6 9H1.5V2C1.5 1.86739 1.55268 1.74021 1.64645 1.64645C1.74021 1.55268 1.86739 1.5 2 1.5ZM1.5 10.5V14C1.5 14.1326 1.55268 14.2598 1.64645 14.3536C1.74021 14.4473 1.86739 14.5 2 14.5H14C14.1326 14.5 14.2598 14.4473 14.3536 14.3536C14.4473 14.2598 14.5 14.1326 14.5 14V10.5H11.163C10.8793 11.0988 10.4315 11.6048 9.87154 11.9591C9.31161 12.3134 8.66261 12.5015 8 12.5015C7.33739 12.5015 6.68838 12.3134 6.12846 11.9591C5.56853 11.6048 5.12069 11.0988 4.837 10.5H1.5ZM0 9V2C0 1.46957 0.210714 0.960859 0.585786 0.585786C0.960859 0.210714 1.46957 0 2 0H14C14.5304 0 15.0391 0.210714 15.4142 0.585786C15.7893 0.960859 16 1.46957 16 2V14C16 14.5304 15.7893 15.0391 15.4142 15.4142C15.0391 15.7893 14.5304 16 14 16H2C1.46957 16 0.960859 15.7893 0.585786 15.4142C0.210714 15.0391 0 14.5304 0 14V9Z"
+			fill="#3858E9"
+		/>
```

Comment:
Ran it through [SVGOMG](https://jakearchibald.github.io/svgomg/) and it made it 38% smaller.

```suggestion
		<path fill="#3858e9" d="M2 1.5h12a.5.5 0 0 1 .5.5v7H10a2 2 0 0 1-4 0H1.5V2a.5.5 0 0 1 .5-.5m-.5 9V14a.5.5 0 0 0 .5.5h12a.5.5 0 0 0 .5-.5v-3.5h-3.34a3.5 3.5 0 0 1-6.32 0zM0 9V2a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2z"/>
```

Reply:
Nice tool, thanks!

Reply:
Reverting, the icon isn't the same.